lib/css/components/_stacks-notices.less

```diff
@@ -29,6 +29,11 @@
         box-shadow: @bs-sm;
         pointer-events: all;
     }
+
+    a:not(.s-btn) { // We don't want to overwrite our button styling
+        color: inherit;
```

Comment:
This is going to apply the color with 021 specificity, which in particular means it's more specific than `.s-link` -- is that intentional?

Reply:
Yeah, currently the `s-link`s look a little goofy in there, so I opted for styling the `a` instead of the `s-link`. We can re-style the `s-link`, but since this is a component I thought it might be nice to not have to add `s-link` to each link, though I'm happy to.

Reply:
That was not my question. You're making it impossible to use `s-link` here.

Reply:
This CSS didn't ship. This is an outdated conversation. Instead of very specific CSS, I added `s-link__underlined` and changed the examples to just use an `s-link s-link__inherit s-link__underlined` :v:

Reply:
I think I just missed pressing the `resolve conversation` button 😛

Reply:
Nah, a conversation attached to a code line is usually marked as outdated when that code line is edited afterwards. Anyway, moot point. All good.